### PR TITLE
Move profile actions to `Settings::ProfileController`

### PR DESCRIPTION
**Pull request by pixeldesu**
Also moves profile picture related things into their own `Settings::ProfilePictureController`.

**Testing:**
* Verify that updating profile info and profile pictures still works

### CODE REVIEW

**Review by raccube — APPROVED** [review]

File: app/views/settings/profile/edit.haml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+          %img.mw-100.mr-3{ src: current_user.profile_header.url(:mobile) }
+        .col-xs-12.mt-3.mt-sm-0.pl-3.pr-3
+          = f.file_field :profile_header, accept: APP_CONFIG[:accepted_image_formats].join(",")
+
+      .row.d-none#profile-header-crop-controls
+        .col-sm-10.col-md-8
+          %strong= t(".adjust.profile_header")
+          %img#profile-header-cropper{ src: current_user.profile_header.url(:web) }
+
+      = f.check_box :show_foreign_themes
```
Comment (nilsding) [inline app/views/settings/profile/edit.haml:27]: not part of this PR, but... why is this here with the pictures?

**Review by nilsding — APPROVED** [review]

File: app/views/settings/profile/edit.haml (hunk 1 above)
Comment (pixeldesu) [reply app/views/settings/profile/edit.haml:27]: Always has been, no idea why it is there really
